### ISSUE TITLE
Ensure proposals continue to fulfil the goals of the W3C

### ISSUE THREAD

**Issue by jwrosewell** [w3c/w3process#477]
If at any time the proposal ceases or looks likely not to be fulfilling the goals of the W3C as defined in the membership agreement, then it can be ruled out of scope. I have provided an analysis on this subject in issue [459](https://github.com/w3c/w3process/issues/459).

This would help to ensure that proposals support the achievement of a competitively neutral standard and thus that liability under competition law should not arise for members and the W3C.

**Comment by chaals** [comment]
This seems redundant with #475

**Comment by dwsinger** [comment]
We have had questions about work and whether it's aligned with our scope, or is in accordance with our values, before, and I expect such debates to happen again. The experience is that those debates were robust, and effective, in illuminating the considerations. 

In specific, if a WG Charter proposes work that is outside the scope of the W3C, or conflicts with its values, we have had, and I expect we would again have, formal objections filed by AC Reps, which triggers a community debate.

If you think further mechanisms are needed, could you be more specific about for what cases, and why, and how?

**Comment by TzviyaSiegman** [comment]
I agree that this seems to duplicate existing issues. All charters have clear scope statements. Perhaps what we are looking for is a clearer understanding of Scope? If Scope is defined too tightly, WGs have no wiggle room when they need to iterate. @jwrosewell would you please explain how tightening scope would achieve neutrality? I fear it would simply make things difficult on the ground.

**Comment by dwsinger** [comment]
see #476 #480 and #478, transfer to AB
